### DISCUSSION TITLE
Duplicates under search calls

### DISCUSSION THREAD

**Discussion by titothefixer** [chuot/rdio-scanner#301] [Q&A]
I have duplicate call detection on and I'm still getting duplicates when I search calls.  It will only play the transmission live once but in search, its there twice. How do I get this to stop?

**Comment by hoffadg** [comment]
I am also having problems with duplicate calls.  I've tried increasing the Duplicate Call Detection Time Frame but it doesn't seem to help.

**Reply by titothefixer** [reply]
Do you have audio conversion on?  Can you test turning it off and seeing if that removes the duplicate for you?

**Comment by chuot** [comment]
What recorder are you using to generate the files and are you recording multiple sites for the same system? This is a known issue with Rdio Scanner v6, but the upcoming v7 will have some additional tricks to detect them.

**Reply by titothefixer** [reply]
I'm using SDRTrunk to record a P25 Phase 2 system. I'm only monitoring 1 site. I'm not sure if it matters but it is a simulcast site.

**Reply by chuot** [reply]
Until v7 is released, try using SDRTrunk's duplicate detection. Never tried it, but saw the feature somewhere in the menus.

**Reply by titothefixer** [reply]
Thank you for your suggestion. It's currently completely on. Both Talkgroup and Radio ID. It does detect the EMS runs that are broadcasted on 3 different channels which Rdio Scanner doesn't. BUT unfortunately there's still duplicates in recordings only. I'll keep my eye out for V7! Thanks again for the assistance.

**Reply by titothefixer** [reply]
I did find the issue.  When Audio conversion is on, it saves both the original AND the converted audio in the same folder.  Both are ingested into search.  Only one is played but both are in search.

**Reply by chuot** [reply]
That is strange because the audio conversion is done in memory and shouldn't create a new file. Unless something changed with FFMPEG.

Is there any error messages at the console output when calls are ingested, something related to FFMPEG ?

**Reply by titothefixer** [reply]
The only error I see is occasionally Rdio can't find a file to remove it.![image](https://user-images.githubusercontent.com/91039132/211936070-86f374a0-ed70-46b3-abb3-b7c85de14273.jpeg)

**Reply by titothefixer** [reply]
I turned it back on to catch the issue I saw last night. It's saying duplicate call rejected but it's still showing up in search. ![image](https://user-images.githubusercontent.com/91039132/211936460-b27faa68-f510-40d5-81d5-20fed507956f.jpeg)
